### ISSUE TITLE
DEFAULT_SERVER_SHUTDOWN_TIMEOUT，This number is too big

### ISSUE THREAD

**Issue by LiZhenNet** [apache/incubator-dubbo#2877]
- [x] I have searched the [issues](https://github.com/apache/incubator-dubbo/issues) of this repository and believe that this is not a duplicate.
- [x] I have checked the [FAQ](https://github.com/apache/incubator-dubbo/blob/master/FAQ.md) of this repository and believe that this is not a duplicate.

### Environment

* Dubbo version: 2.6.2
* Operating System version: mac os
* Java version: 1.8

### Steps to reproduce this issue

com.alibaba.dubbo.common.Constants#DEFAULT_SERVER_SHUTDOWN_TIMEOUT，
I think this number is too big，2~3 second  is enough。

Pls. provide [GitHub address] to reproduce this issue.

### Expected Result

What do you expected from the above steps？

### Actual Result

What actually happens?

If there is an exception, please attach the exception trace:

```
Just put your stack trace here!
```

**Comment by carryxyh** [comment]
Can u give some reason?

**Comment by LiZhenNet** [comment]
> Can u give some reason?
> 10 seconds is an insurance time. If you think 10s is too big, please adjust dubbo.service.shutdown.wait

I know  it can change value by change the dubbo.service.shutdown.wait, But, 10 second is not  a appropriate value.  

In 2.5.x , it will sleep 10 second between Registry destroy and Protocol

**Comment by carryxyh** [comment]
Since this sleep occurs when the service goes offline, what problems will it cause when you sleep longer?

**Comment by carryxyh** [comment]
In my opinion, the difference between 10s or 5s is not very big. I can't decide which is better. Each has its own advantages and disadvantages. 5s is better? maybe.

**Comment by LiZhenNet** [comment]
> In my opinion, the difference between 10s or 5s is not very big. I can't decide which is better. Each has its own advantages and disadvantages. 5s is better? maybe.

It will make the application shutdown slow  in old version。
This config in 2.6.x  ，it's role that make the request which is excuting  return success. In general,  a request  should not execute more than 5 seconds。

**Comment by carryxyh** [comment]
I agree with what you said. But personally think that the difference between fast closing and slow closing is not very big. 
Should we modify it? Maybe. @zonghaishang Can u give some ideas?

**Comment by LiZhenNet** [comment]
> I agree with what you said. But personally think that the difference between fast closing and slow closing is not very big.
> Should we modify it? Maybe. @zonghaishang Can u give some ideas?

I have pull a request to talk about it  https://github.com/apache/incubator-dubbo/pull/2901
